src/include/OpenImageIO/fmath.h

```diff
@@ -137,12 +137,12 @@ pow2rounddown (int x)
 
 
 
-/// Round up to the next whole multiple of m.
-///
-inline int
-round_to_multiple (int x, int m)
+/// Round value up to the next whole multiple.
+/// For example, round_to_multiple(7,10) returns 10.
+template <typename V, typename M>
+inline V round_to_multiple (V value, M multiple)
```

Comment:
I wanted it to be the same type, but then you have:

    uint32_t x = ...;
    uint32_t y = round_to_multiple (x, 4);

4 is an int, so you get an error.

We DO have unit tests for this, though they studiously use the same type. I'll set up additional unit test cases that would have failed before and now succeed.